server/routes/index.js

```diff
@@ -15,5 +16,6 @@ router.get('/', (req, res) => {
 
 router.use('/auth', userRouter);
 router.use('/oauth', socialRouter);
+router.use('/book', bookRouter);
```

Comment:
I think `books` would be more appropriate.

Reply:
I agree.